notebooks/4_plotting.ipynb

```diff
@@ -0,0 +1,512 @@
+{
```

Comment:
<p>Maybe ax instead of of ax_1 (since there is only one) ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>I did this to overcome the potential confusion some new users might have with <code>.plot(ax=ax)</code> which _feels_ like magic to them without realizing the left hand side is the method parameter and the right hand side is the ax object passed as argument. I propose to use <code>axs</code> instead of <code>ax</code></p><p> </p>